unionstore/memdb_test.go

```diff
@@ -699,14 +698,12 @@ func mustGet(t *testing.T, buffer *MemDB) {
 }
 
 func TestKVGetSet(t *testing.T) {
-	defer testleak.AfterTestT(t)()
```

Comment:
whether we just replace with `defer goleak.VerifyNone(t)` in this pass?

Reply:
Adding in each Test is a bit tedious, I am going to add goleak in each package's `TestMain`. Please wait for my subsequent PR.

Reply:
Well. Given leak tests don't in used actually. I'm ok with this.